WalletWasabi.Gui/Controls/WalletExplorer/CoinListViewModel.cs

```diff
@@ -595,42 +623,13 @@ public void OnCoinIsSelectedChanged(CoinViewModel cvm)
 
 		public void OnCoinStatusChanged()
 		{
-			SetCoinJoinStatusWidth();
+			RefreshStatusColumnWidth();
 		}
 
-		public void OnCoinUnspentChanged(CoinViewModel cvm)
+		public void OnCoinUnspentChanged(CoinViewModel _)
```

Comment:
we don't need the parameter here.

Reply:
Yes, that is why I wrote _. This is a call from the CoinViewModel and has a fixed method footprint. The parameter must be defined.

Reply:
@molnard 
> The parameter must be defined.

I tried to remove it and I didn't get any error.